### ISSUE TITLE
Documentation suggestion

### ISSUE THREAD

**Issue by mcampsall** [invertase/react-native-firebase#178]
> 2) Link RNFirebase
> Unfortunately, due to the fact that Firebase is much easier to setup using Cocoapods, react-native link is not recommended as it is not customisable enough for our needs and we have had numerous problems reported.

This section is misleading for those of us who see the title: **Link RNFirebase** and then the `react-native link` is the next thing my eye is drawn to, and as I am conditioned, I typed that in terminal and did not read anything else. I know I should always read all the instructions, but the pattern is usually run the terminal command and go to the next step. 

My suggestion is to either change the title (which reads as a step command) or remove the back-ticks from "react-native-link" so it doesn't read like all the other instructions. Or, just remove this section, as it is just general information about the reasons behind why the following step is the way it is.

The only reason i noticed this was _not_ what i was supposed to do, was because I have 'RCTBridgeModule.h' file not found errors and came back to troubleshoot

**Comment by DavideValdo** [comment]
I saw docs about installing without CocoaPods at all, why where they removed?

I'm trying to use this module along with react-native-navigation by wix and getting "Native module cannot be null" error on iOS. Installed this with CocoaPods into a blank RN project, all working good, then followed wix's docs on the .xcworkspace to link their module manually and app broke.

I think my best bet would be to link both modules by hand, without CocoaPods.

**Comment by chrisbianca** [comment]
@mcampsall Fair point, I will tweak the documentation to make this more clear.

@DavideValdo They were removed as they were causing lots of issues as people were trying to get started.  I will look to add them back in this week once I have worked through the process myself and can document it properly.

Can I just ask, are you still planning on using Firebase itself through Cocoapods?  What error are you getting once you've linked react-native-navigation manually?

**Comment by DavideValdo** [comment]
@chrisbianca 
Hello Chris,
yes I have designed my application relying on this module. I referenced this problem in #153, it seems like I was able to link the Firebase SDK since [FIRApp configure] provides good console feedback.  Then React Native loads and I get a "Native Module Cannot Be Null" error, I think this is related to the wrapper itself.

The application is really a toy class I wrote to test out wix's rnn and rnfirebase's behaviour alongside. On Android everything seems to be working fine.
